### DISCUSSION TITLE
Error: unknow-flag: --file

### DISCUSSION THREAD

**Discussion by goldingdamien** [evilmartians/lefthook#622] [Q&A] [answered]
Running below in Git Bash is leading to error below.
It seems like it should work according to the documentation: https://github.com/evilmartians/lefthook/blob/master/docs/usage.md#lefthook-run

`./node_modules/.bin/lefthook run pre-commit --commands my-test --file folder1/test/check.test.ts --file folder1/test/check.test.tsx`
>
Error: unknown flag: --file

**Comment by mrexox** [comment]
Hey! What lefthook version do you use?

**Reply by goldingdamien** [reply]
"lefthook": "1.5.0"

**Reply by mrexox** [reply, marked as answer]
`--file` option was added in 1.6.0 version. You can upgrade, and it must work fine.

**Reply by goldingdamien** [reply]
@mrexox Oh, sorry, my mistake. I'll make sure to update. Thanks.
